extensions/whatsapp/package.json

```diff
@@ -12,24 +12,17 @@
   "license": "MIT",
   "commands": [
     {
-      "name": "open-chat",
-      "title": "Open Chat",
-      "subtitle": "WhatsApp",
-      "description": "Open quickly added WhatsApp chats",
-      "mode": "view"
-    },
-    {
-      "name": "add-chat",
-      "title": "Add Chat",
-      "subtitle": "WhatsApp",
-      "description": "Add WhatsApp chats to open them later",
+      "name": "open-in-whatsapp",
+      "title": "Open In WhatsApp",
+      "subtitle": "Open any contact in whatsapp",
+      "description": "Open any contact in whatsapp by searching for their name or phone number.It autosaves the contacts in the background.",
       "mode": "view"
     },
-    {
-      "name": "add-existing-group",
-      "title": "Add Existing Group",
-      "subtitle": "WhatsApp",
-      "description": "Add existing WhatsApp groups to open them later",
+    { 
+      "name": "refresh",
+      "title": "Refresh Contacts",
+      "subtitle": "Refresh contacts",
+      "description": "Refresh contacts",
```

Comment:
I wonder if we could do better than a refresh command. Why not call the `getContacts` function inside a `useCachedPromise` from `@raycast/utils` within the `Open Chat` command? This way, we don't need a separate command or to store this information manually.